pkg/cmd/step/git/credentialhelper/step_git_credentialhelper.go

```diff
@@ -0,0 +1,159 @@
+package credentialhelper
+
+import (
+	"os"
+
+	"github.com/jenkins-x/jx/pkg/auth"
+	"github.com/jenkins-x/jx/pkg/cmd/opts/step"
+	"github.com/jenkins-x/jx/pkg/gits/credentialhelper"
+	"github.com/pkg/errors"
+
+	"github.com/jenkins-x/jx/pkg/cmd/helper"
+
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	optionGitHubAppOwner = "github-app-owner"
+)
+
+// StepGitCredentialHelperOptions contains the command line flags
+type StepGitCredentialHelperOptions struct {
+	step.StepOptions
+
+	GitKind        string
```

Comment:
I think we would not need these flags in this case. They would not be passed anyways, right?

Reply:
Yeah, am hoping I can get rid of these with a bit of a refactor

Reply:
I think I know how. Maybe we can have a chat once I am home. Using the path matching might work out quite nicely and we even git rid of these flags which would be another win really. 

What's about doing the fix in the old command and just focus on the required changes to fix the issue, opposed to also extract a new command?

Reply:
we probably need an issue in this repo as well.